debian/libspandsp3-dev.install

```diff
@@ -0,0 +1,5 @@
+debian/tmp/usr/include/spandsp3/*/spandsp.h
+debian/tmp/usr/include/spandsp3/spandsp
+debian/tmp/usr/lib/*/spandsp3/libspandsp3.so
```

Comment:
we shouldn't be stuffing libs in a different dir

Reply:
this is related to `we dont want to change libdir`, can be removed when we solve pkg-config issue